### ISSUE TITLE
Date Limit form elements need accessible labels

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#323]
Flagged by axe DevTools:
> **Form elements should have a visible label:** Ensure that every form element has a visible label and is not solely labeled using hidden labels, or the title or aria-describedby attributes

Labels ("Date from" and "Date to") are visible and visually adjacent to the text inputs, but they need to be associated with the text inputs programmatically as well.

Related to issue #322.

# Suggested fix

1. Use `aria-labelledby="#{input-element-id}"` on the label elements in addition to the existing `for="#{input-element-id}"`attribute.
2. While we're at it: change `aria_label="..."` to `aria-label="..."` with hyphen instead of an underscore

**Comment by erinchambers** [comment]
Assigning to @karindalziel for later delegation
